src/components/FormValidator.js

```diff
@@ -77,10 +80,15 @@ class FormValidator {
   enableValidation() {
     this._formElement.addEventListener("submit", (e) => {
       e.preventDefault();
-      this.disableButton();
     });
     this._setEventListeners();
-    this.disableButton();
+  }
+
+  resetValidation() {
+    this._inputElements.forEach((inputElement) => {
+      this._checkInputValidity(inputElement);
```

Comment:
`      this._checkInputValidity(inputElement);
`

Please, use `_hideInputError` to 100% reset validation otherwise there are errors under the inputs though the user hasn't typed anything yet


```js

    this._inputElements.forEach((inputElement) => {
      this._hideInputError(inputElement);  // clear errors 100% 
    });
```